src/components/MuiVirtualizedTable/MuiVirtualizedTable.js

```diff
@@ -814,6 +852,7 @@ class MuiVirtualizedTable extends React.PureComponent {
                     flexDirection: 'column',
                     height: '100%',
                 }}
+                className={this.props.className}
             >
```

Comment:
why aren't all the static styles 'sxed' in this part

Reply:
I migrated makeStyle/useStyle to emotion but left everything else as is. Can be refactored in a separate PR